docs/build.gradle

```diff
@@ -213,6 +213,9 @@ task generateJavaDocs(type: Javadoc) {
     options.links("https://docs.oracle.com/en/java/javase/17/docs/api/")
     options.links("https://docs.opencv.org/4.x/javadoc/")
     options.addStringOption("tag", "pre:a:Pre-Condition")
+    options.addStringOption("Xmaxerrs","9999")
+    options.addStringOption("Xmaxwarns","9999")
+    options.addBooleanOption("Xdoclint/package:-edu.wpi.first.math.proto,-edu.wpi.first.math.controller.*,-edu.wpi.first.math.geometry.*,-edu.wpi.first.math.kinematics.*,-edu.wpi.first.math.system.plant.*,-edu.wpi.first.math.trajectory.proto", true)
```

Comment:
Is there a way to specify a file of these rather than list them all here?

Reply:
I don't think so. I think the best we can do is concatenating an array.